packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/subagent/react_tools.py

```
@@ -0,0 +1,1351 @@
+            "GRANT",
+            "REVOKE",
+        ]
+        for kw in forbidden:
+            if sql_upper.startswith(kw):
+                return json.dumps(
+                    {
+                        "chunks": [
+                            {
+                                "output_type": "text",
+                                "content": (
+                                    f"安全限制: 不允许执行 {kw} 语句，"
```

Ruff (0.15.21)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)